onmt/trainer.py

```diff
@@ -260,13 +259,6 @@ def train(self,
             # UPDATE DROPOUT
             self._maybe_update_dropout(step)
 
-            if self.gpu_verbose_level > 1:
-                logger.info("GpuRank %d: index: %d", self.gpu_rank, i)
-            if self.gpu_verbose_level > 0:
-                logger.info("GpuRank %d: reduce_counter: %d \
-                            n_minibatch %d"
-                            % (self.gpu_rank, i + 1, len(batches)))
```

Comment:
Are we 100% confident this can't be useful anymore in some debugging scenario?

Reply:
let's remove it for now, I am keeping the option in and when I will re-address the multi-node we'll check again.